go/vt/sqlparser/parse_test.go

```diff
@@ -611,27 +611,27 @@ func TestValid(t *testing.T) {
 	}, {
 		input: "select /* GE false */ 1 from t where a >= false",
 	}, {
-		input: "select * from t order by a collate utf8_general_ci",
+		input:  "select * from t order by a collate utf8_general_ci",
```

Comment:
the precommit hooks asked me to run gfmt on this since I modified the file -- see the real change on line 803 below